src/components/SingleDatePicker.jsx

```diff
@@ -161,6 +162,11 @@ class SingleDatePicker extends React.PureComponent {
     }
 
     this.isTouchDevice = isTouchDevice();
+
+    // We manually set event because React has not implemented onFocusIn/onFocusOut.
+    // Keep an eye on https://github.com/facebook/react/issues/6410 for updates
+    // We use "blur w/ useCapture param" vs "onfocusout" for FF browser support
+    this.container.addEventListener('blur', this.onFocusOut, true);
```

Comment:
As mentioned offline, I think it would be preferable and simpler to not have this blur functionality but instead rely on the tab callbacks in the DayPicker

Reply:
@majapw This is from @TaeKimJR's implementation before I took on this project, but the weird situation here is that the only context we have from the browser `onTab` is what element we were focused on *before* the `onTab`? `document.activeElement` seems to be reset to `body` during the event cycle w/ React. I think `e.relatedTarget` may be what I'm looking for, but still need to test that theory. The only solution I've found thus far is quite hacky.